app/views/ips/index.html.erb

```diff
@@ -12,7 +12,9 @@
     </div>
     <div class="govuk-grid-column-one-third text-right">
       <% if current_user.can_manage_locations?(current_organisation) %>
-        <%= link_to "Add a location", new_location_path, class: "govuk-button govuk-!-margin-bottom-0", role: "button", draggable: "false", "data-module" => "govuk-button" %>
+        <%= link_to "Add a location", new_location_path, class: "govuk-button", role: "button", draggable: "false", "data-module" => "govuk-button" %>
```

Comment:
should we have this grouped horizontally?  That is what we do  here https://admin.staging.wifi.service.gov.uk/ips for instance.